applications/GeoMechanicsApplication/tests/test_thermal_element/test_micro_climate/README.md

```diff
@@ -0,0 +1,50 @@
+# Test Cases for Micro-Climate Boundary Condition
+
+**Author:** [Mohamed Nabi](https://github.com/mnabideltares)
+
+**Source files:** [Micro Climate Boundary Condition](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_thermal_element/test_micro_climate)
+#
+# Case Specification
+We employed eight test cases to encompass a range of potential scenarios and thoroughly evaluate all functionalities related to the micro-climate within the code. These test cases, each varying in the constant required from the user, are detailed in the table below.
+
+| Simulation| Describtion            | a1  | a2  | a3   | Q   | Smax   |
```

Comment:
It might be a good idea to link to the microclimate condition documentation, such that we can find the same variables there and what they mean and use the same values:
```suggestion
| Simulation| Describtion            | $a_1$  | $a_2$  | $a_3$   | $Q_f$   | Smax   |
```

I could not find the Smax in the original docs, but that's probably me